bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -1562,9 +1561,8 @@ spec:
                                 on an Ingress object).
                               properties:
                                 describedObject:
-                                  description: CrossVersionObjectReference contains
-                                    enough information to let you identify the referred
-                                    resource.
```

Comment:
I don't know why the descriptors are full of this exact sentence. Regenerating manifests removed it, but it seems we could upload a wrong/outdated version of the manifests and CI didn't complain.